src/fu-pending.c

```diff
@@ -153,17 +174,29 @@ fu_pending_add_device (FuPending *pending, FuDevice *device, GError **error)
 	g_debug ("FuPending: add device %s", fu_device_get_id (device));
 	statement = sqlite3_mprintf ("INSERT INTO pending (device_id,"
 							  "state,"
+							  "flags,"
 							  "filename,"
+							  "firmware_id,"
 							  "display_name,"
 							  "provider,"
```

Comment:
Unrelated, but I just noticed this is still provider.  I think you should separate from this request update this table to rename to plugin.

Reply:
Agreed. If we end up renaming the database I'll do the switcharoo then.